spyder/utils/encoding.py

```diff
@@ -111,6 +113,17 @@ def get_coding(text):
             # sometimes we find a false encoding that can result in errors
             if codec in CODECS:
                 return codec
+
+    # Falback using chardet
+    if is_binary_string(text):

```

Comment:
Ok, I agree with you guys, thanks for the clarification :-)

@rlaverde, please don't forget to fix the little typo pointed out by @goanpeca on the line above this line :-)